lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_git_helper.rb

```diff
@@ -41,9 +41,9 @@ def self.commit_version_bump(include_deliverfile: true, include_metadata: true)
         def self.localize_project
           Action.sh("cd #{ENV['PROJECT_ROOT_FOLDER']} && ./Scripts/localize.py")
```

Comment:
While we're in here, WDYT about adding a check that these environment variables are present before starting this method? IMHO it would make anything that's missed far more obvious without having to dig into the source. Just thinking basically a runtime assertion (until we can refactor this to use proper arguments and stop assuming certain `ENV` keys exist, anyway...)

Reply:
Oh yeah! I think we've been too relaxed with `ENV` vars and are paying the price now that we're changing things.

Reply:
Partially addressed in 0239b8d. I didn't go down the full refactoring with DI route to prioritizie shipping the fix.